### PR TITLE
[Blocked] => Remove react-tracking.d.ts

### PR DESCRIPTION
**Pull request by damassi**
DefinitelyTyped PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35425

The above must be released before this can be merged. Looks like `Decorator` and `Options` aren't exported from the definition file.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 778a3dafe148e19eeebcc89758b2442e6fa9d4f1

https://deploy-preview-2411--artsy-reaction.netlify.com

File: src/Apps/Order/Components/PaymentPicker.tsx [hunk 1]
```diff
@@ -371,7 +371,8 @@ const PaymentPickerWithInnerRef: React.SFC<
   <PaymentPicker ref={innerRef} {...props as any} />
 )
 
-export const PaymentPickerFragmentContainer = createFragmentContainer(
+// FIXME: After deleting `react-tracking.d.ts, this needed to be cast due to HOC
+export const PaymentPickerFragmentContainer: any = createFragmentContainer(
```
Comment (damassi) [inline src/Apps/Order/Components/PaymentPicker.tsx]: @ds300 - After removing the local `d.ts` file this was needed. HOC's are tricky and I couldn't figure out how to fix.

Comment (ds300) [reply src/Apps/Order/Components/PaymentPicker.tsx]: Yeah doesn't look easy with how the `track` decorator is typed now.

We could fix that upstream, but for now would be good to turn this `any` into something less dangerous. Could do that by adding a cast inline after the `track()(...)` expression:

image

**Comment by alloy** [comment]
DT PR is merged 👍

**Comment by damassi** [comment]
Updated types to `@types/react-tracking@5.0.4` and addressed @ds30's comment (so _that's_ how you do it 😄)

**Comment by artsyit** [comment]
:rocket: PR was released in v16.11.6 :rocket:

**Pull request merged**